packages/cli/package.json

```diff
@@ -81,7 +81,8 @@
         "lodash": "^4.17.21",
         "moment": "^2.29.1",
         "normalize-path": "^3.0.0",
-        "puppeteer": "^5.5.0",
+        "playwright": "^1.19.2",
```

Comment:
where do we use playwright?

Reply:
It's an optional dependency from Apify but it's referenced in Apify's typings, so the build breaks if we don't install it ourselves